doc/classes/Vector2i.xml

```diff
@@ -124,6 +124,14 @@
 		<constant name="ONE" value="Vector2i(1, 1)">
 			One vector, a vector with all components set to [code]1[/code].
 		</constant>
+		<constant name="MIN" value="Vector2i(-2147483648, -2147483648)">
+			Min vector, a vector with all components equal to [code]INT_MIN[/code]. Can be used as a negative integer equivalent of [constant Vector2.INF].
+			[b]Note:[/b] The components are not equal to the min value of [int], which is 64-bit unlike vectors.
+		</constant>
+		<constant name="MAX" value="Vector2i(2147483647, 2147483647)">
+			Max vector, a vector with all components equal to [code]INT_MAX[/code]. Can be used as an integer equivalent of [constant Vector2.INF].
```

Comment:
Docs should be changed too to `INT32_MIN/MAX`. Though this is C terminology and not necessarily familiar to readers. but if we merge #68999, we'll be able to turn them into proper references to our definitions.